pkg/startup/v7/data/common/etc/fluentd/fluent.conf

```diff
@@ -28,14 +28,14 @@
 </filter>
 <match **>
   @type forward
-  send_timeout 60s
-  recover_wait 10s
-  hard_timeout 60s
-
+  buffer_type memory
+  buffer_queue_full_action block
```

Comment:
What's the consequence here if the agent encounters a lot of backpressure? Is it possible for this to OOM the host or block indefinitely?

Reply:
Hrm, good question, I had maintained the same settings @mjudeikis had in the previous version of the logging stack, but according to https://docs.fluentd.org/v/0.12/buffer#handling-queue-overflow I feel like either throwing an exception or dropping old chunks would be better... The arch is like this: journald -> fluentd -> memory buffer -> mdsd, what do you all think?